react/StackContainer.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC, createContext } from 'react'
+
+import useCumulativeHeightState from './modules/useCumulativeHeightState'
+import { Positions } from './typings'
+
+interface StackContext {
+  position?: Positions
+  onResize(height: number): void
+  stickOffset: number
+}
+
+interface Props {
+  position: Positions
+}
+
+export const StackContext = createContext<StackContext>({
+  onResize: () => {},
+  stickOffset: 0,
+})
+
+const StackContainer: FC<Props> = ({ children, position = Positions.TOP }) => {
```

Comment:
```suggestion
/** This component is used to accumulate the height of multiple StickLayout blocks,
 * so that they get stacked instead of appearing in front of each other */
const StackContainer: FC<Props> = ({ children, position = Positions.TOP }) => {
```

ou algo similar